robot.py

```
@@ -35,6 +35,23 @@ def reply_restart_bot(message: RobotMessage):
     os.execl(sys.executable, sys.executable, *sys.argv)
 
 
+@command(tokens=["配置重载", "reload_conf"], permission_level=PermissionLevel.ADMIN)
+def reply_reload_conf(message: RobotMessage):
+    Constants.reload_conf()
+    message.reply("已重载配置文件")
+    Constants.log.info("[obot-core] 已重载配置文件")
+
+
+@command(tokens=["对话场景ID", "chat_scene_id"])
+def reply_reload_conf(message: RobotMessage):
```

Ruff (0.12.2)

Redefinition of unused `reply_reload_conf` from line 39

(F811)